vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -57,17 +57,22 @@ class MSBuildTools {
     newSuccess(`Found Solution: ${slnFile}`);
     newInfo(`Build configuration: ${buildType}`);
     newInfo(`Build platform: ${buildArch}`);
-
+    if (target) {
+      newInfo(`Build target: ${target}`);
+    }
     const verbosityOption = verbose ? 'normal' : 'minimal';
-    const errorLog = path.join(process.env.temp, `msbuild_${process.pid}.err`);
+    const errorLog = path.join(
+      process.env.temp,
+      `msbuild_${process.pid}${target ? '_' + target : ''}.err`,
+    );
     const args = [
       `/clp:NoSummary;NoItemAndPropertyList;Verbosity=${verbosityOption}`,
       '/nologo',
       '/maxCpuCount',
       `/p:Configuration=${buildType}`,
       `/p:Platform=${buildArch}`,
       '/p:AppxBundle=Never',
-      '/bl',
+      `/bl${target ? `:${target}.binlog` : ''}`,
```

Comment:
I updated teh CI upload logic to upload `*.binlog` and added also `*.binlog` to .gitignore
#4971